dnf/module/module_base.py

```diff
@@ -641,6 +641,9 @@ def _get_info(self, module_specs):
                         for mod_require, stream in require_dict.items():
                             req_set.add("{}:[{}]".format(mod_require, ",".join(stream)))
                 lines["Requires"] = "\n".join(sorted(req_set))
+                demodularized = modulePackage.getDemodularizedRpms()
+                if demodularized:
+                    lines["Demodularized rpms"] = "\n".join(modulePackage.getDemodularizedRpms())
```

Comment:
Optimize.
Is there any reason to call `modulePackage.getDemodularizedRpms()` again and not use the result of the previous call stored in `demodularized`?

Reply:
Correct, I am fixing it

Reply:
Fixed